src/org/zaproxy/zap/extension/zest/resources/Messages.properties

```diff
@@ -122,6 +122,22 @@ zest.dialog.assign.label.string			= String:
 zest.dialog.assign.label.strprefix		= Prefix String:
 zest.dialog.assign.label.strpostfix		= Postfix String:
 zest.dialog.assign.label.variable		= Variable Name:
+zest.dialog.assign.label.filterByElement							= Filter by element name:
+zest.dialog.assign.label.filterByElementName						= Element name:
+zest.dialog.assign.label.filterByAttribute							= Filter by attribute name:
+zest.dialog.assign.label.filterByAttributeName						= Attribute name:
+zest.dialog.assign.label.filterByAttributeValue						= Attribute value (RegExp):
+zest.dialog.assign.label.filteredElementsReversed					= Reverse filtered elements:
+zest.dialog.assign.label.filteredElementsIndex						= Index of the element:
+zest.dialog.assign.label.filteredElementsSelector					= Select elements:
+zest.dialog.assign.label.filteredElementsSelectorAttributeName		= Select value of attribute:
```

Comment:
I believe all these labels should be in title caps.
zaproxy/zaproxy#2000
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#text-titles-menu-items-options-and-tool-tips